src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+$border-color: #f3f3f3;
+$product-card-width: 200px;
+$main-fonts: 'Roboto';
+$main-text-color: #060b35;
+$second-text-color: #616070;
+$button-color: #00acdc;
+$border-radius: 5px;
+
 body {
   margin: 0;
 }
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $product-card-width;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+  align-items: center;
+  justify-content: center;
+
+  &__image {
+    background-image: url('../images/imac.jpeg');
```

Comment:
According to the checklist, images should be in the format of .png, .jpg, or .jpeg. The image 'imac.jpeg' should be checked to ensure it is in the correct format and renamed if necessary to adhere to the format guidelines.